Makefile

```diff
@@ -1,94 +1,71 @@
-DOCKER          = docker
-DOCKER_COMPOSE  = docker-compose
+SELF_DIR := $(dir $(lastword $(MAKEFILE_LIST)))
+PHP_SERVICE := docker-compose exec php sh -c
```

Comment:
I don't understand why you remove DOCKER_COMPOSE to add PHP_SERVICE. The first one is much more used than the second one.
I think you could have kept the alias DOCKER_COMPOSE and add the PHP_SERVICE alias like `$(DOCKER_COMPOSE) exec php sh -c`

Reply:
Long story short:
- `$(DOCKER_COMPOSE)` = 17 characters
- `docker-compose` = 14 characters

From my point of view, there is no need to use a variable for this binary. It's more useful for the `php` service as it's a real shortcut.

Reply:
Good for me, thank you for the explanation